electron-app/src/components/settings/api-keys/ExchangeSettings.vue

```diff
@@ -120,12 +132,39 @@ export default class ExchangeSettings extends Vue {
   showPassphrase: boolean = false;
 
   connectedExchanges!: string[];
+  krakenAccountTypes = ['starter', 'intermediate', 'pro'];
+  selectedKrakenAccountType = '';
+
+  mounted() {
+    this.selectedKrakenAccountType = this.settings.krakenAccountType;
```

Comment:
If you go with the getter this should not be needed anymore

Reply:
It is. You need it in order to populate the dropdown when you load the exchange settings of kraken, to show what the currently selected account type is.

Reply:
I think since there is never anything pre-selected on first page visit, if you've linked `v-model` in the "account type" select correctly once someone clicks "kraken" you shouldn't need to do an additional mounted() call, it should auto-populate like the inputs for the key/secret. Will try to tinker today unless kelsos knows right away.